src/SourceTerm/Manager.cpp

```diff
@@ -97,8 +95,36 @@ class index_sort_by_value
     }
 };
 
+void seissol::sourceterm::computeMInvJInvPhisAtSources(Eigen::Vector3d const& centre,
```

Comment:
is` centre` a good name for this variable? (I guess this is not the tetra center?).

Reply:
You could interpret it as the centre of the (sub-)fault the point source represents.

Reply:
ok I understand the logic (same name as in the NRF reader).

Reply:
The bigger problem is the weird British spelling here ;)